src/main/java/redis/clients/jedis/StreamGroupInfo.java

```diff
@@ -0,0 +1,61 @@
+package redis.clients.jedis;
+
+import java.io.Serializable;
+import java.util.Map;
+
+public class StreamGroupInfo implements Serializable {
+
+  public final static String NAME = "name";
+  public final static String CONSUMERS = "consumers";
+  public final static String PENDING = "pending";
+  public final static String LAST_DELIVERED = "last-delivered-id";
+  public static final String GROUP_INFO = "groups";
+
+
+  /*private final String name;
+  private final long consumers;
+  private final long pending;
+  private final String lastDeliveredId;*/
```

Comment:
May be, keep these variables? Along with the map, as well as in other classes.
The variables will ease implementation. The map would help during changes in Redis.
WDYT?

Reply:
Do you mean to keep them together with getters? So classes provide both `getXYZInfo()` method returning map and getters for already known fields? I like this idea.

Reply:
@MichalCho 

> Do you mean to keep them together with getters? So classes provide both getXYZInfo() method returning map and getters for already known fields?

YES.

Reply:
ok, the getters are there. And constants can be used to access map elements.
Thanks